pkg/ioutils/scheduler.go

```diff
@@ -0,0 +1,6 @@
+// +build !gccgo
+
+package ioutils
+
+func call_scheduler_if_necessary() {
```

Comment:
I know this is minor, but seems like most of the codebase has stayed away from python-like function/variable naming and instead uses camelCase--so seems this should be `callSchedulerIfNecessary`?